### PR TITLE
Add arm64 builds

### LINKED ISSUE #342: Arm Image
**Is this a request for help?**:

---

**Is this a BUG REPORT or FEATURE REQUEST?** (choose one):

Feature Request:

It would be great if there was a linux/arm64 version of the image in particular for use on M1 macs. quay.io supports multiarch images now.

**Version of Helm and Kubernetes**:

**What happened**:

**What you expected to happen**:

**How to reproduce it** (as minimally and precisely as possible):

**Anything else we need to know**:

### PR DESCRIPTION
**Pull request by tylerauerbeck**
Signed-off-by: Tyler Auerbeck <tauerbec@redhat.com>

**What this PR does / why we need it**:
This PR adds the building and publishing of arm64 images to quay when a release is published.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #342 

**Special notes for your reviewer**:

This PR adds the publishing of arm64 images as well as amd64 images when a release is published. Because the same `goreleaser` functionality is used in the `ci` workflow, we needed to add the buildx and qemu actions there as well.

When a release is cut, there will now be a few images pushed to quay. For example, given a release of v3.4.5, you would end up with:

- chart-testing:latest-amd64
- chart-testing:latest-arm64
- chart-testing:v3.4.5-amd64
- chart-testing:v3.4.5-arm64
- chart-testing:v3.4.5
- chart-testing:latest

The last two would be the ones that would be useful to users and would direct them to the appropriate arch (and their manifests would just reference the underlying versions). Just wanted to call this out so it wasn't a surprise when this started to happen.

### CODE REVIEW

File: .goreleaser.yml [hunk 1]
```diff
@@ -54,3 +53,31 @@ dockers:
...
+      - --label=org.opencontainers.image.licenses=Apache-2.0
+      - --label=org.opencontainers.image.source=https://github.com/helm/chart-testing
+      - --label=org.opencontainers.image.authors=The Helm Authors
+    extra_files:
+      - etc/chart_schema.yaml
+      - etc/lintconf.yaml
+docker_manifests:
+- name_template: quay.io/helmpack/chart-testing:latest
+  image_templates:
+  - quay.io/tauerbec/chart-testing:latest-amd64
```
Comment (paulczar) [inline .goreleaser.yml:78]: shouldn't these be `quay.io/helmpack/chart-testing` ?

File: .goreleaser.yml [hunk 2]
```diff
@@ -54,3 +53,31 @@ dockers:
...
+  - quay.io/tauerbec/chart-testing:latest-arm64
+- name_template: quay.io/helmpack/chart-testing:{{ .Tag }}
+  image_templates:
+  - quay.io/tauerbec/chart-testing:{{ .Tag }}-amd64
```
Comment (paulczar) [inline .goreleaser.yml:82]: shouldn't these be `quay.io/helmpack/chart-testing` ?

**Review by paulczar — CHANGES_REQUESTED** [review]

File: .goreleaser.yml (hunk 2 above)
Comment (tylerauerbeck) [reply .goreleaser.yml:82]: 🤦🏻 They definitely should. Thought I grabbed these. Let me get this fixed up.

Comment (tylerauerbeck) [reply .goreleaser.yml]: @paulczar Fixed.

File: .goreleaser.yml (hunk 1 above)
Comment (tylerauerbeck) [reply .goreleaser.yml]: @paulczar Fixed.

**Review by paulczar — APPROVED** [review]

**Comment by paulczar** [comment]
hey @scottrigby can you confirm this looks good to you too ?

**Review by scottrigby — COMMENTED** [review]
looks like I need to catch up on goreleaser options!
Looked over docs, this all looks correct AFAICT. Let's trigger Ci…

**Comment by scottrigby** [comment]
Also thank you @tylerauerbeck 😄

**Comment by scottrigby** [comment]
CI failed. Can we check why? I'm not in a place to do that right now

**Comment by tylerauerbeck** [comment]
@scottrigby Yepp. On my list to dig into today. I'll let you know what I find.

**Comment by tylerauerbeck** [comment]
@scottrigby @paulczar Sorry for the delay on this one, but had some time to go back and take a look. I've been able to run this successfully on my own branch, which you can see here:

release: https://github.com/tylerauerbeck/chart-testing/actions/runs/1498224439
ci: https://github.com/tylerauerbeck/chart-testing/actions/runs/1498109036

Published images from the release workflow: https://quay.io/repository/tylerauerbeck/chart-testing?tab=tags

**Comment by cpanato** [comment]
@tylerauerbeck please sign the dco

**Comment by tylerauerbeck** [comment]
@cpanato Good catch. Signoff is there now.

**Review by cpanato — APPROVED** [review]
thanks for that

**Comment by cpanato** [comment]
@scottrigby will merge ok? :)

**Pull request merged**